### ISSUE TITLE
Add cliff sensor support

### ISSUE THREAD

**Issue by jacobperron** [AutonomyLab/create_autonomy#22]
At the very least the driver can publish a Boolean message signalling whether or not a cliff is detected.  
Alternatively, can also publish a custom message similar to the [Bumper](https://github.com/AutonomyLab/create_autonomy/blob/indigo-devel/ca_msgs/msg/Bumper.msg) message, that contains the raw signal strengths as well as the Boolean cliff signals for all four sensors.

**Comment by betaupsx86** [comment]
Yep definitely. I added Cliff custom messages while working with the driver. Just let me know if you are open to contributions

**Comment by jacobperron** [comment]
@betaupsx86 Definitely open for contributions :)

**Comment by Dcruise546** [comment]
@betaupsx86 Is there any update on the cliff custom message? I am trying to use the sensor for an experimental docking mechanism. Any help will be much appreciated!

**Comment by girvenavery2022** [comment]
Hi @jacobperron, I just finished this feature and tested it on my create2. I can confirm it works and can open a pr to contribute it!

**Comment by jacobperron** [comment]
@girvenavery2022 There is already a PR adding this feature (#93). Sorry, I've just been slow at getting around to testing it.
